account/accounts.go

```diff
@@ -387,40 +336,17 @@ func (m *Manager) Accounts() ([]gethcommon.Address, error) {
 	m.mu.RLock()
 	defer m.mu.RUnlock()
 
-	am, err := m.geth.AccountManager()
+	_, err := m.geth.AccountManager()
```

Comment:
@gravityblast  you can remove this line right?

can you clarify why we don't read other addresses from manager now?

Reply:
this methods is called from `web3.eth.getAccounts()`.
For now we never had child accounts so we always returned the only address we have.
Now we added more accounts, also to let people having more addresses for privacy reasons. 
I wouldn't return all of them to a dapp. We also discussed to have a default address option in the future to allow the user decide which address to expose. So I thought to do it like this.
What do you think about it?

Reply:
thanks, that makes sense

so you are saying that web3.eth.getAccounts is using manager.Accounts? can you show where it is called? if we will use our own instance of keystore will it affect that (https://github.com/status-im/status-go/pull/1546) ?

Reply:
@dmitryn here we define the handler for `eth_accounts`

https://github.com/status-im/status-go/blob/develop/api/backend.go#L425